build-system/app.js

```diff
@@ -313,7 +312,7 @@ function proxyToAmpProxy(req, res, mode) {
     body = body
         // Unversion URLs.
         .replace(/https\:\/\/cdn\.ampproject\.org\/rtv\/\d+\//g,
-            'https://cdn.ampproject.org/')
+        'https://cdn.ampproject.org/')
```

Comment:
Is this indentation right?

Reply:
Per our existing rules, it is right. First continuation line is indented by 4 spaces, following lines use the same indentation. This was done automatically by `--fix`, and wasn't a manual change. If this seems wrong to you, I could look at adding a more refined rule and rerunning the linter in `--fix` mode across all our files.